### DISCUSSION TITLE
Cadenas IV Parte B

### DISCUSSION THREAD

**Discussion by damianperezz** [INGCOM-UNRN-P1/cursada-2023#189] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141683317/2e15ba4f-ed91-4de9-9c3b-3880bf95a93f)
En el prototipo de funcion dice que toma a un arreglo, esto deberia tomar un char* cadena? Y el tamaño se debe pasar o solo se pasaria la cadena? ya que en el prototipo de la concatenacion dinamica no se pasaron los tamaños
Y lo que devuelve la funcion deberia ser tambien un char* ?

**Comment by martinvilu** [comment, marked as answer]
https://github.com/orgs/INGCOM-UNRN-P1/discussions/188
